Source/Core/Core/PowerPC/Interpreter/Interpreter_LoadStorePaired.cpp

```diff
@@ -124,11 +120,6 @@ float Interpreter::Helper_Dequantize(const u32 _Addr, const EQuantizeType _quant
 	case QUANTIZE_S16:
 		fResult = static_cast<float>((s16)Memory::Read_U16(_Addr)) * m_dequantizeTable[_uScale];
 		break;
-
-	default:
```

Comment:
Then why aren't those labeled and added to the EQuantizeType enum, or rather, why test an enum argument in a switch if something can lie outside of it? That makes no sense.